src/qrl/core/BlockHeader.py

```diff
@@ -122,6 +166,15 @@ def create(blocknumber: int,
                prev_timestamp: int,
                hashedtransactions: bytes,
                fee_reward: int):
+        """
+        Intended to be called by Block.create(), which will calculate these params
```

Comment:
No need to talk how it is called by. Idea of documentation is talk about what it does, not how it is called.

Reply:
Well, what it does is obvious enough in this case, so I'm deleting the comment